One instance of the exception is active, and a second instance of the exception is pending.<BR>&nbsp;&nbsp;&nbsp; Only asynchronous exceptions can be active and pending. Any synchronous exception is either inactive, pending, or active